lnbits/core/services.py

```diff
@@ -533,3 +534,21 @@ async def send_data(self, message: str, item_id: str):
 
 async def websocketUpdater(item_id, data):
     return await websocketManager.send_data(f"{data}", item_id)
+
+
+async def switch_to_voidwallet() -> None:
+    WALLET = get_wallet_class()
+    if WALLET.__class__.__name__ == "VoidWallet":
+        return
+
+    set_wallet_class("VoidWallet")
+    setattr(settings, "lnbits_backend_wallet_class", "VoidWallet")
```

Comment:
- is there a difference between the above line and the more simple?
  - `settings.lnbits_backend_wallet_class = "VoidWallet"`
- might be, not sure, just asking

Reply:
there is none, but setattr() is required if the key you want to set is a variable, in those case both works, but i changed it to the simple one.